docs/docs/features/proposals.md

```diff
@@ -75,18 +75,33 @@ Minimal example:
 ```
 
 ## ChangeRewardDenomProposal
-:::tip
-`ChangeRewardDenomProposal` will only be accepted on the provider chain if at least one of the denomsToAdd or denomsToRemove fields is populated with at least one denom. Also, a denom cannot be repeated in both sets.
-:::
 
 Proposal type used to mutate the set of denoms accepted by the provider as rewards.
 
+:::tip
+`ChangeRewardDenomProposal` will only be accepted on the provider chain if at least one of the `denomsToAdd` or `denomsToRemove` fields is populated with at least one denom. Also, a denom cannot be repeated in both sets.
+:::
+
 Minimal example:
 ```js
+{
+  "title": "Add uatom as a reward denom",
+  "description": "Here is more information about the proposal",
+  "denomsToAdd": ["uatom"],
+  "denomsToRemove": []
+}
+```
+
+:::tip
+Besides native provider denoms (e.g., `uatom`), please use the `ibc/*` denom trace format. 
+For example, use `ibc/0025F8A87464A471E66B234C4F93AEC5B4DA3D42D7986451A059273426290DD5`, 
```

Comment:
Dumb question: Could you explain here how to find the ibc denom of an asset on the hub? specifically what commands to run to figure it out, e.g. how you knew that untrn is ibc/...... on the hub?

Reply:
I just saw, later you explain how to get the base denom from an ibc denom. I think the other direction is also relevant, i.e. how do you find the ibc denom when you know the base denom.

Reply:
See https://github.com/cosmos/interchain-security/pull/1489/commits/78f3c9ec81992c7b382c2ecbf74ae3e6890ed37f